R/calcIO.R

```diff
@@ -225,5 +237,5 @@ calcIO <- function(subtype = c("input", "output", "output_biomass", "trade",
   }
 
   return(list(x = reminditems, weight = NULL, unit = "EJ",
-              description = "IEA SE Output Data based on 2017 edition of IEA World Energy Balances"))
+              description = "IEA SE Output Data based on 2022 edition of IEA World Energy Balances"))
```

Comment:
would be different when we will update the latest version

Reply:
see me comment above. this is accurate for the default, aka the inputdata generation. when we update to latest version, we will have to update here, too (and in many other places)

Reply:
o.k., hope we will remember